packages/mds-provider/api.ts

```diff
@@ -327,15 +320,17 @@ function api(app: express.Express): express.Express {
     return Promise.all(promises)
   }
 
-  // FIXME add pagination?
+  // TODO validate query params, 
```

Comment:
This is a very different TODO than the original FIXME... Can anyone weigh in a bit?

Reply:
Well it's got pagination in the method body, so it's been added.

Reply:
Sounds like it's ready to be removed.

Reply:
Looking at the body, there's param validation actually, so removing this todo.